include/zephyr/net/mqtt.h

```diff
@@ -356,6 +356,22 @@ struct mqtt_sec_config {
 	/** Indicates the list of security tags to be used for the session. */
 	const sec_tag_t *sec_tag_list;
 
+#if defined(CONFIG_MQTT_LIB_TLS_USE_ALPN)
+	/**
+	 * Pointer to array of char array indicate the ALPN protocol name
+	 * May be NULL to skip ALPN protocol negotiation
+	 */
+	const char **alpn_protocol_name_list;
+
+	/**
+	 * Indicate number of ALPN protocol name in alpn protocol name list.
+	 *
```

Comment:
Please remove empty line.
```suggestion
	 * Indicate number of ALPN protocol name in ALPN protocol name list.
```